app/controllers/user_roles_controller.rb

```diff
@@ -15,6 +15,15 @@ def update
     end
   end
 
+  def remove
+    if @user_role.project.leader? current_user
+      @user_role.update active: false
+      redirect_to members_project_path(@user_role.project), notice: t('.success')
+    else
+      redirect_to root_path, alert: t('.fail')
+    end
```

Comment:
Aqui daria pra usar um `guard clause`?

Reply:
Eu vi que em outro controller, foi implementado o `set_project` da seguinte maneira:

```ruby
def set_project
    @project = if params[:project_id]
                 Project.find(params[:project_id])
               else
                 @invitation.project
               end
end
```

Não tenho certeza se assim é uma melhor prática, mas se for, daria pra vocês implementarem assim no controller de `user_roles`. Desse jeito, daria pra chamar o `set_project` e o `authorize_leader` andes dessa action `remove`

Reply:
Ficou bem legal esse mesmo!